tools/cpp/runfiles/runfiles_src.cc

```diff
@@ -79,6 +80,21 @@ bool ends_with(const string& s, const string& suffix) {
   return s.rfind(suffix) == s.size() - suffix.size();
 }
 
+std::vector<std::string> split(const std::string& s, const std::string& sep) {
```

Comment:
If `sep` is always `"/\\"`, we could inline that value, remove the parameter and call the function `split_path`.